components/threading/global_threads_link/global_threads_link.tsx

```diff
@@ -36,6 +37,10 @@ const GlobalThreadsLink = () => {
     const unreadsOnly = useSelector(isUnreadFilterEnabled);
     const someUnreadThreads = counts?.total_unread_threads;
 
+    const closeRHS = useCallback(() => {
+        dispatch(suppressRHS);
+    }, []);
+
```

Comment:
I'm not really sure I get why do we want to memoize this with no changing arguments, isn't it the same as
```suggestion
    const closeRHS = () => {dispatch(suppressRHS)};
```